### PR TITLE
Set a range for line spacing

### LINKED ISSUE #27912: Script Text disappear when the line spacing is set to a big number
Godot version 3.1

OS/device including version:
MacOs Mojave 10.14.3
McBook Air 13 (2015)

Issue description: 
Script Text disappear when the line spacing is set to a big number and messed up when is set to a negative number. The line spacing should have a limit range.

**Steps to reproduce:**
Editor-> Editor Settings -> Text Editor -> Theme -> Line spacing
Default line spacing
Default line spacing (6)

Negative line spacing
Negative line spacing (-12)

Big line spacing
Big line spacing (1000)

### PR DESCRIPTION
**Pull request by gsegovia2018**
Fixes #27912

### CODE REVIEW

**Comment by volzhs** [comment]
i use 0 for it.
6 for minimum size is too big for me.

**Comment by akien-mga** [comment]
I agree that if 6 is the default value, it shouldn't be the minimum, there are always people who want less spacing. I guess even some low negative numbers might still be relevant.

You could use "0,50,1,or_lesser,or_greater" to set a range without preventing people from going beyond.

File: editor/editor_settings.cpp [hunk 1]
```diff
@@ -406,6 +406,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("text_editor/theme/color_theme", "Adaptive");
 	hints["text_editor/theme/color_theme"] = PropertyInfo(Variant::STRING, "text_editor/theme/color_theme", PROPERTY_HINT_ENUM, "Adaptive,Default,Custom");
 	_initial_set("text_editor/theme/line_spacing", 6);
+    hints["text_editor/theme/line_spacing"] = PropertyInfo(Variant::INT, "text_editor/theme/line_spacing", PROPERTY_HINT_RANGE, "6,50,1");
```
Comment (akien-mga) [inline editor/editor_settings.cpp]: Indentation should be done with tabs.

**Comment by akien-mga** [comment]
Force pushed an update to fix the invalid merge commit (which readded a previously removed setting).

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**